### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by KoushikKintali**
card

### CODE REVIEW

File: components/alert/alert.html [hunk 1]
```diff
@@ -21,16 +22,32 @@
...
+                <h3>Components</h3>
+                <span id="components-expand-more" class="material-icons-outlined display-block" onclick="componentsExpandMore()">
+                    expand_more
+                </span>
+                <span id="components-expand-less" class="material-icons-outlined display-none" onclick="componentsExpandLess()">
+                    expand_less
+                </span>
+            </div>
+            <ul id="components-nav-items" class="components-nav-items">
                 <a href="../avatar/avatar.html"><li class="side-nav-items">Avatar</li></a>
```
Comment (sanathkumar26700) [inline components/alert/alert.html]: add styling like color, :active, :visited to anchor tags

File: components/card/card.html [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        </aside>
+        <div class="doc-content">
+            <h4>Card Component</h4>
+            <hr>
+            <section>
+                <div class="components">
+                    <h5>Cards with badges</h5>
+                    <div class="preview-block">
+                        <div class="card">
+                            <img class="card-img" src="https://picsum.photos/id/0/250">
```
Comment (anuj-kumary) [inline components/card/card.html]: Always give **alt** attribute to img tag it helps to screen reader

**Review by MonikaJhingan — APPROVED** [review]
Consistent use of rem should be there which u can do in later projects otherwise the code looks good to be merged.

**Comment by KoushikKintali** [comment]
> Consistent use of rem should be there which u can do in later projects otherwise the code looks good to be merged.

I didn't get this, can u please elaborate @MonikaJhingan

**Pull request merged**